static/app/utils/window/useIsFullscreen.tsx

```diff
@@ -0,0 +1,20 @@
+import {useEffect, useState} from 'react';
+import screenfull from 'screenfull';
+
+export default function useIsFullscreen() {
```

Comment:
Can we add some comments describing what this hook is for (and maybe how it differs from `useFullscreen`)? +same thing for the other hook